src/test/runTest.ts

```diff
@@ -1,6 +1,6 @@
 import * as path from 'path';
 import { downloadAndUnzipVSCode, resolveCliArgsFromVSCodeExecutablePath, runTests } from '@vscode/test-electron';
-const cp = require('child_process');
+const { spawnSync } = require('node:child_process');
```

Comment:
in PR desc you have mentioned `void to use require()` but it seems it is still used?

Reply:
effectively, why is it not reported by eslint then? oO
I will ook to it